tools/pot/openvino/tools/pot/api/README.md

```diff
@@ -81,7 +13,15 @@ The base class for all DataLoaders.
 by index. 
 
 All subclasses should override `__len__()` function, which should return the size of the dataset, and `__getitem__()`, 
-which supports integer indexing in range of 0 to `len(self)`
+which supports integer indexing in range of 0 to `len(self)`. `__getitem__()` method can return data in on of the possible format:
+```
+(data, annotation)
+```
+or
+```
+(data, annotation, metadata)
+```
+`data` is the input which is passed to the model at inderence so that it should be properly preprocessed. `data` can be either `numpy.array` object or dictionary where key is the name of the model input and value is `numpy.array` which corresponds to this input. The format of `annotation` should correspond to the expectations of the `Metric` class. `metadata` is an optional field that can be used to store additional information required for post-processing.
```

Comment:
```suggestion
`data` is the input that is passed to the model at inference so that it should be properly preprocessed. `data` can be either `numpy.array` object or dictionary where the key is the name of the model input and value is `numpy.array` which corresponds to this input. The format of `annotation` should correspond to the expectations of the `Metric` class. `metadata` is an optional field that can be used to store additional information required for post-processing.
```